geo-bool-ops-benches/benches/boolean_ops.rs

```diff
@@ -80,7 +83,7 @@ fn run_complex<T: Measurement>(c: &mut Criterion<T>) {
         group.bench_with_input(BenchmarkId::new("geo::intersects", steps), &(), |b, _| {
```

Comment:
```suggestion
        group.bench_with_input(BenchmarkId::new("geo::relate", steps), &(), |b, _| {
```

Probably good to keep the output in line with what's being benchmarked - though TBH I'm not sure why we're benchmarking the `intersects` method in the overlay / bool-ops benchmarks.